library/asn1write.c

```diff
@@ -82,8 +82,12 @@ int mbedtls_asn1_write_len( unsigned char **p, unsigned char *start, size_t len
         *--(*p) = 0x83;
         return( 4 );
     }
-
+/*
+* relevant only for 64 bit platforms. on 32 bit platforms, size_t should always be 32 bits
```

Comment:
Style: Could you please align the `*` characters as in:
```
/*
 *
 */
```
Also, please make the first letter uppercase and ensure that the line is no longer than 80 characters as per the mbed TLS style guide: https://tls.mbed.org/kb/development/mbedtls-coding-standards